jsonrpc/http.go

```diff
@@ -62,6 +75,7 @@ func (h *HTTP) ServeHTTP(writer http.ResponseWriter, req *http.Request) {
 	}
 
 	req.Body = http.MaxBytesReader(writer, req.Body, MaxRequestBodySize)
+	h.requests.Inc()
```

Comment:
Why are we only incrementing and not decrementing when we have responded to the request?

Reply:
We are not trying to keep track of concurrent requests. We are simply counting incoming requests so that we can calculate the rate from that.